### PR TITLE
Mustache templates

### PR DESCRIPTION
**Pull request by RobertHebel**

### CODE REVIEW

File: tools/js/data.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
+const fs = require('fs');
+const markTwain = require('mark-twain');
+const jsonML = markTwain(fs.readFileSync('README.md').toString());
+const { parseJsonMLFactory } = require('../../tools/build');
+
+const data = {
+  slider: [
+    {
+      src: 'assets/img/carousel/carousel-image-2.jpg',
+      alt: '',
```
Comment (adam-golab) [inline tools/js/data.js]: Could you provide `alt` values here?

File: tools/mustache/contact.mustache [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{{#offices}}
+    <p>
+        <span>{{name}}</span>
+        <span>{{address}}</span>
+        <span>{{city}}</span>
+    </p>
+{{/offices}}
```
Comment (adam-golab) [inline tools/mustache/contact.mustache]: Missing newline

File: tools/mustache/index.mustache [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <li>
+            <button class="tabs-nav__button" data-category="teammates">Teammates projects</button>
+        </li>
+    </ul>
+    <div class="tabs-content__bg">
+        {{>tabs}}
+    </div>
+</section>
+
+{{>footer}}
```
Comment (adam-golab) [inline tools/mustache/index.mustache]: Missing newline

File: tools/mustache/tabs.mustache [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<ul class='tabs__content container'>
+    {{#tabsContent}}
+        <li class='tabs-content__item' data-category={{category}}>
+            <a href={{& href}}>
+                <h3 class='tabs-content__header'>{{name}}</h3>
+                <p>{{desc}}</p>
+            </a>
+        </li>
+    {{/tabsContent}}
+</ul>
```
Comment (adam-golab) [inline tools/mustache/tabs.mustache]: Missing newline

File: tools/mustache/contact.mustache [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{{#offices}}
+    <p>
```
Comment (adam-golab) [inline tools/mustache/contact.mustache]: Why did you use 4 spaces for indentation? Could you change indentation in all mustache files to 2 spaces?

File: tools/mustache/tabs.mustache (hunk 4 above)
Comment (RobertHebel) [reply tools/mustache/tabs.mustache]: added

File: tools/mustache/contact.mustache (hunk 5 above)
Comment (RobertHebel) [reply tools/mustache/contact.mustache]: changed

File: tools/mustache/index.mustache (hunk 3 above)
Comment (RobertHebel) [reply tools/mustache/index.mustache]: added

File: tools/mustache/contact.mustache (hunk 2 above)
Comment (RobertHebel) [reply tools/mustache/contact.mustache]: added

File: tools/js/data.js (hunk 1 above)
Comment (RobertHebel) [reply tools/js/data.js]: added

File: package.json [hunk 6]
```diff
@@ -14,24 +14,24 @@
   "license": "MIT",
   "homepage": "https://brainhubeu.github.io",
   "devDependencies": {
-    "eslint": "^4.11.0",
-    "eslint-config-brainhub": "^1.6.3",
-    "jest": "^21.2.1",
```
[the remark this answers is not in the document]
Comment (RobertHebel) [reply package.json]: added

File: gulpfile.js [hunk 7]
```diff
@@ -7,7 +7,8 @@ const browserify = require('gulp-browserify');
 const uglify = require('gulp-uglify');
 const babel = require('gulp-babel');
 const eslint = require('gulp-eslint');
-
+const mustache = require('gulp-mustache');
+const data = require('./tools/js/data').data;
```
[the remark this answers is not in the document]
Comment (RobertHebel) [reply gulpfile.js]: changed

**Pull request merged**